pkg/iostreams/iostreams.go

```diff
@@ -202,6 +206,10 @@ func (s *IOStreams) CanPrompt() bool {
 	return s.IsStdinTTY() && s.IsStdoutTTY()
 }
 
+func (s *IOStreams) GetNeverPrompt() bool {
```

Comment:
Added for testing purposes.

Reply:
Yep, definitely could. My thought was that there is additional logic `s.IsStdinTTY() && s.IsStdoutTTY()` in `CanPrompt` and adding this accessor function was a more direct test since I really just wanted to check that `SetNeverPrompt(true)` had been called in the test.